### ISSUE TITLE
List items not programmatically defined as list items

### ISSUE THREAD

**Issue by bbsmooth** [freeCodeCamp/freeCodeCamp#57352]
### Describe the Issue

Don't know how this one slipped by me. The five tech companies listed at the top of the home page are in an unordered list, but each company is not in an actual list item. Any automated accessibility checker will report this as an issue. The only permitted direct children of a list are `li` elements.

```
<ul>
  <svg>...</svg>
  <svg>...</svg>
  <svg>...</svg>
  <svg>...</svg>
  <svg>...</svg>
</ul>
```

### Affected Page

https://www.freecodecamp.org/

### Steps to Reproduce

1. Go to 'https://www.freecodecamp.org/'
2. Open the list of five tech companies in the DOM inspector
3. Notice that there are five `svg` children in the `ul`

### Expected behavior

Each `svg` element in the list should be wrapped in an `li` element. 

```
<ul>
  <li><svg>...</svg></li>
  <li><svg>...</svg></li>
  <li><svg>...</svg></li>
  <li><svg>...</svg></li>
 <li><svg>...</svg></li>
</ul>
```

### System

N/A

**Comment by ojeytonwilliams** [comment]
Hey Bruce, I suspect you haven't seen it before because it's part of an AB test!  

https://github.com/freeCodeCamp/freeCodeCamp/blob/6fc7a04611a03f3f3cdb5eb0b300f1b67cad28a6/client/src/pages/index.tsx#L46

LandingTopB being the offender.

Is the approach in LandingTop okay?

https://github.com/freeCodeCamp/freeCodeCamp/blob/6fc7a04611a03f3f3cdb5eb0b300f1b67cad28a6/client/src/components/landing/components/landing-top.tsx#L51-L69

**Comment by bbsmooth** [comment]
@ojeytonwilliams Ahh, that explains it. Yes, LandingTop is fine. I don't think these absolutely have to be in a list. It's just that if they are, as in LandingTopB, then they need to be wrapped in `li`.

**Comment by ojeytonwilliams** [comment]
Perfect, thanks Bruce. In that case, I'll open it for contribution.

**Comment by Sneaky321** [comment]
Hi! Is this issue still open for contribution? I’d love to help if it’s available.
